assets/elasticsearch/ingest.py

```
@@ -55,15 +63,46 @@ def traverse_data(collection_name):
 
 
 def ingest_dataset():
     es = Elasticsearch("http://localhost:9202", request_timeout=1200, retry_on_timeout=True)
 
+    if sys.argv[3] != "no_logsdb":
+        template_body = {
+            "index_patterns": [collection_name],
+            "template": {
+                "settings": {
+                    "index": {
+                        "mode": "logsdb"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)